source/beamline/BioXAS/BioXASZebraPulseControl.cpp

```diff
@@ -200,8 +203,10 @@ void BioXASZebraPulseControl::setInputValue(int value)
 
 void BioXASZebraPulseControl::setEdgeTriggerValue(int value)
 {
-	if (!edgeTriggerControl_->withinTolerance(double(value)))
-		edgeTriggerControl_->move(double(value));
+	if (edgeTriggerValue_ != value) {
+		edgeTriggerValue_ = value;
```

Comment:
You are right @helfrij.  And I do like the idea of changing the name to preference.  I think it makes it clearer.